src/atomsWithQuery.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  ApolloClient,
+  OperationVariables,
+  QueryOptions,
+  ApolloQueryResult,
+} from '@apollo/client'
+import { Getter, WritableAtom } from 'jotai'
+import { clientAtom } from './clientAtom'
+import { createAtoms } from './common'
+
+type QueryArgs<
+  Variables extends object = OperationVariables,
+  Data = any
+> = QueryOptions<Variables, Data>
+
+type AtomWithQueryAction<
+  Variables extends object = OperationVariables,
+  Data = any
+> = {
+  type: 'refetch'
+  opts?: Partial<QueryArgs<Variables, Data>>
```

Comment:
This is not used, so it should be removed. (unsupported)

Reply:
Oh, sure! Thanks